src/python/grpcio/tests/unit/framework/foundation/_logging_pool_test.py

```diff
@@ -29,6 +29,8 @@
 
 """Tests for grpc.framework.foundation.logging_pool."""
 
+from __future__ import division
```

Comment:
Your right, this change (or the following one) wasn't necessary.  I just did a grep to find all instances of division across our code base to enforce a standard across the codebase.  If you have strong feelings about this, I can remove the import.